sdk/maps/azure-maps-search/README.md

```diff
@@ -60,8 +60,8 @@ Once completed, set the values of the client ID, and client secret of the AAD ap
 from azure.identity import DefaultAzureCredential
 from azure.maps.search import SearchClient
 credential = DefaultAzureCredential()
-client_id = os.environ.get("CLIENT_ID")
-authentication_policy = AzureKeyCredential(os.environ.get("SUBSCRIPTION_KEY"))
+client_id = os.environ.get("AZURE_CLIENT_ID")
+authentication_policy = AzureKeyCredential(os.environ.get("AZURE_CLIENT_SECRET"))
```

Comment:
This is different from samples/search.py and can cause confusion for users. Should we also use this in samples/search.py? Or suggest to use "snippet" to generate the code here.